modules/renderer/map.js

```diff
@@ -311,7 +311,7 @@ export function rendererMap(context) {
     function resetTransform() {
         if (!transformed) return false;
 
-        surface.selectAll('.radial-menu').interrupt().remove();
+        surface.selectAll('.edit-menu').interrupt().remove();
```

Comment:
Well, so we will end up having two of these (one for radial, other for edit).
I am thinking if about all the cleaning that would need to be done for a major release ..